core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/uri/ComponentURI.java

```diff
@@ -96,7 +96,7 @@ public String identifier() {
 
 	@JsonIgnore
 	public final boolean isUnspecified() {
-		return CodeSystem.uri(TerminologyRegistry.UNSPECIFIED).equals(resourceUri());
+		return TerminologyRegistry.UNSPECIFIED.equals(resourceUri().getResourceId());
```

Comment:
Hmm, this means that the `UNSPECIFIED` value becomes a special resource ID value, we might need to add a validation rule to the resource create API.

Reply:
Historically (in 7.x) we checked it this way, then it became wrapped into `CodeSystem.uri(...)` during the rewrite.

Reply:
Yes, I know, the validation logic should be available since the introduction of this special value. I'll approve the PR and merge it. @apeteri please create a spin-off ticket that describes the issue of having this special ID as resource ID, thanks!